README.md

```diff
@@ -34,32 +34,24 @@ When the reader has completed this Code Pattern, they will understand how to:
 
 ## Included Components
 
-* [IBM Model Asset Exchange](https://developer.ibm.com/code/exchanges/models/): A place for developers to find and use
+* [IBM Model Asset Exchange (MAX)](https://developer.ibm.com/code/exchanges/models/): A place for developers to find and use
 free and open source deep learning models.
```

Comment:
could you move the acronym change further up in the readme, where the term "Model Asset Exchange" is first mentioned? And remove it here - it only needs to be done once.

Reply:
will do